src/sajari.php

```diff
@@ -0,0 +1,442 @@
+<?php
+
+namespace Sajari;
+
+require dirname(__FILE__) . '/vendor/autoload.php';
+require dirname(__FILE__) . '/proto/doc.php';
+require dirname(__FILE__) . '/proto/query.php';
+require dirname(__FILE__) . '/document.php';
+require dirname(__FILE__) . '/search.php';
+
+interface Opt
```

Comment:
@tbillington unless I'm missing changes in PHP these days, per PSR0 autoloading, every class and interface should be in its own file under a directory that matches the namespace. So WithEndpoint should be found at `src/Sajari/WithEndpoint.php`. Note: a newer PSR autoloading spec (PSR4?) lets you omit the `Sajari` directory I believe so the files can be found at `src/WithEndpoint.php`. Best check those specs though as it's been a while for me.

cc @dhowden